src/api/spec/factories/notification.rb

```diff
@@ -1,8 +1,42 @@
 FactoryBot.define do
   factory :notification do
+    type { 'Notification' }
     event_type { 'FakeEventType' }
```

Comment:
This should be one of the real event types I think.

Reply:
Let me check

Reply:
That's not an event type. That's a `Notification` type.

Reply:
Ah you were talking about the `event_type` field... I thought you were talking about the `type` field :man_facepalming:

Reply:
I've set a real event type.